free-3.md

```diff
@@ -7,44 +7,126 @@ good way to meet their needs.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+Grading in computer science becomes a difficult and involved task with large
+classes. Many times a grader will leave the same comment on multiple students
+code and have to specify where a students code went wrong and why they deducted
+points on a rubric.
+
+It would be useful if we adopted the DRY approach to grading and removed the
+need to create redundent information. There should be a consice way of saying
+on a rubric the line(s) of code that caused you to deduct points.
+
+Natural language rubrics are not easily analyzed beyond scores in large categories.
+By creating a formal markup, we could better summarize behaviors of students and
+graders.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Graders in CS are usually pretty comfortable with programming, so a language
+would fit in. Furthermore, creating a formal DSL will require the graders to
+be more precise in their feedback and allow professors to analyze larger grading
+trends.
```

Comment:
I also imagine that once you formalize the grading process it might be easier to hook in static analysis and auto-grading tools (i.e. recognize non-idiomatic code, poor formatting, etc)